blueocean-dashboard/src/main/less/connection-lost.less

```diff
@@ -0,0 +1,30 @@
+.errorscreen {
+    &.connection-lost {
+        background-color: @errorscreen-bg-color;
+    }
+    &.connection-ok {
+        background-color: @blueocean-blue;
+    }
+
+    &.connection-lost, &.connection-ok {
+        /*
+         * Lets ensure that the lost and ok messages are the same
+         * width Vs jumpy message box resizes.
+         */
+        .message-box {
+            min-width: 30em;
+            max-width: 30em;
+        }
+
+        /*
+         * For some reason, we needed to set the font-weight on the
+         * h3 used in the errorscreen when disconnected. If not, it
+         * appears more bold than it does if the same screen is
+         * shown when connected, plus when compared to the 404 screen.
+         * Weird :(
+         */
+        h3 {
+            font-weight: 400;
+        }
```

Comment:
@kzantow you mean the video? If so, that's because I have a specific `font-weight` tweak in there to make it look "kinda" similar ... they're not the same though ... different fonts.

I suppose this is really a small issue that we can come back to if we find an answer to it.